examples/AutomatedSummaryReport/main.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/io;
+import ballerinax/slack;
+
+// Define the Slack API token
+configurable string token = ?;
+
+// Define a record type to hold channel information
+type ChannelType record {
+    string id;
+};
+
+// Define a record type to hold the response structure of the channels list
+type Channels record {|
+    boolean ok;
+    ChannelType[] channels;
+|};
+
+// Define a record type to hold text messages
+type TextType record {
+    string text;
+};
+
+// Define a record type to hold the response structure of the conversation history
+type History record {
+    boolean ok;
+    TextType[] texts;
+};
+
+// Initialize the Slack client with the provided token
+final slack:Client slack = check new Client({
+    auth: {
+        token: value
+    }
+});
+
+public function main() returns error? {
+    // Fetch the list of channels
+    json channelResponse = check slack->/conversations\.list();
```

Comment:
Can't this and other similar methods support data binding instead of returning json? @NipunaRanasinghe

Reply:
Discord OpenAPI itself hasn't provided names for most of the schemas (98 out of 174 resources), so manually sanitising all the similar cases would've taken much more time. 

@TharmiganK @lnash94 IIRC we discussed about this case sometimes back and, I hope we'll be able to come up with some approach to generate a meaningful named responses for such cases from the openAPI tool's side?

Reply:
This is a bug, created an issue here: https://github.com/ballerina-platform/ballerina-library/issues/6698